packages/vue/src/components/templates/internalData.js

```diff
@@ -0,0 +1,86 @@
+export const accountData = {
+  firstName: "John",
+  lastName: "Dog",
+  email: "johndog@email.com",
+  password: "a*23Et",
+  shipping: [
+    {
+      firstName: "John",
+      lastName: "Dog",
+      streetName: "Sezame Street",
+      apartment: "24/193A",
+      city: "Wroclaw",
+      state: "Lower Silesia",
+      zipCode: "53-540",
+      country: "Poland",
+      phoneNumber: "(00)560 123 456",
+    },
+    {
+      firstName: "John",
+      lastName: "Dog",
+      streetName: "Sezame Street",
+      apartment: "20/193A",
+      city: "Wroclaw",
+      state: "Lower Silesia",
+      zipCode: "53-603",
+      country: "Poland",
+      phoneNumber: "(00)560 123 456",
+    },
+  ],
+  orders: [
+    ["#35765", "4th Nov, 2019", "Visa card", "$12.00", "In process"],
+    ["#35766", "4th Nov, 2019", "Paypal", "$12.00", "Finalised"],
+    ["#35768", "4th Nov, 2019", "Mastercard", "$12.00", "Finalised"],
+    ["#35769", "4th Nov, 2019", "Paypal", "$12.00", "Finalised"],
+  ],
```

Comment:
OrderHistory component requires orders prop as an object. It can be changed here or in the component. Which is better?
![Screenshot from 2021-09-28 12-04-21](https://user-images.githubusercontent.com/32042425/135067250-e2a8d9f1-a2a3-4a1a-a1fa-0c21196951f2.png)